### PR TITLE
Document the primary restriction of this chart

### PR DESCRIPTION
**Pull request by yuvipanda**
Fixes https://github.com/2i2c-org/binderhub-service/issues/95

### CODE REVIEW

**Comment by yuvipanda** [comment]
The restrictions section I hope is useful in deciding 'does this functionality go into this chart?'

**Comment by yuvipanda** [comment]
I've reworded the background section to a 'history' section instead, as this chart is less 'experimental' now

File: README.md [hunk 1]
```diff
@@ -7,32 +7,44 @@ The binderhub-service is a Helm chart and guide to run BinderHub (the Python
...
+
+## Restrictions
+
+To prevent a recurrance of the issues with the existing binderhub chart, the following
+restrictions are in place for any work on this chart:
+
+> There will not be a _direct_ dependency on a JupyterHub. We can provide documentation on
+> how to set this chart up next to a JupyterHub, but we will not provide a JupyterHub
+> directly (via a [helm dependency](https://helm.sh/docs/chart_best_practices/dependencies/))
+> or otherwise.
```
Comment (GeorgianaElena) [inline README.md:45]: Great rule of thumb!!!

**Review by GeorgianaElena — APPROVED** [review]
This is looking very good @yuvipanda. Thank you!

**Pull request merged**